Webex Connect Flows/v3.0/Template/README.md

```diff
@@ -0,0 +1,50 @@
+# webexcc-digital-channels
+
+This repo provides the template workflows for Webex Contact Center and IMI integration.
+
+## Workflows Overview
+The following workflow templates are included :
+
+1. **Media Specific Workflows** : These flows need to be imported once per asset of the specific channel.
+    * **Facebook Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated Facebook page.
+    * **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured customer chat widget.
```

Comment:
Should we say, widget configured with Live Chat Form like below statement

Reply:
Sure